components/datepicker/WeekDay.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const WeekDay = ({ weekday, className, localeUtils, locale, size }) => {
```

Comment:
Functional components do not have a built-in comparison (`shouldComponentUpdate`). Do we want that?

Reply:
Valid point. I've changed this to a `class` extending `PureComponent`.

 :ok: by @driesd on 30/05/2018 14:31:31